src/index.ts

```diff
@@ -93,7 +91,7 @@ class WebpackManifestPlugin implements WebpackPluginInstance {
       const hook = !NormalModule.getCompilationHooks
         ? compilation.hooks.normalModuleLoader
         : NormalModule.getCompilationHooks(compilation).loader;
-      hook.tap(hookOptions, normalModuleLoader);
+      hook.tap(hookOptions, normalModuleLoader as any);
```

Comment:
The use of `as any` here will be a blocker for this PR

Reply:
so without casting it won't work

Reply:
look at the types before your changes (what's on master), and compare to what it is with your changes, adjust from there.

Reply:
before my change it has import
```ts
// @ts-ignore
import NormalModule from 'webpack/lib/NormalModule';
```
so NormalModule was any. 
You suggest to replicate previous types, so I'll need to 
```ts
    const NormalModule = compiler.webpack.NormalModule as any;
```

Reply:
I think upcasting from object to more specific type inside normalmodule hook is better, more types are preserved/checked

Reply:
> second try. 
By this I meant that I've made second fix, removed as any.